source/clear-linux/guides/maintenance/repair-failed-install.rst

```diff
@@ -0,0 +1,71 @@
+.. _repair-failed-install:
+
+Repair a failed installation of |CL-ATTR|
+#########################################
+
+This guide explains how to repair a failed installation of |CL| using a live
```

Comment:
I prefer "broken installation" instead of "failed"

Reply:
+1 title should be "broken" since the system could need repair at any point in its lifecycle, not just installation. The body can include the words broke, failed, and corrupt to get the point across well.